packages/material-ui-lab/src/internal/svg-icons/ErrorOutlined.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import createSvgIcon from './createSvgIcon';
+
+/**
+ * @ignore - internal component.
+ */
+export default createSvgIcon(
+  <path d="M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20,12C20,16.41 16.41, 20 12,20M12,2C6.47,2 2,6.47 2,12C2,17.53 6.47,22 12,22C17.53,22 22,17.53 22,12C22,6.47 17.53, 2 12,2M14.59,8L12,10.59L9.41,8L8,9.41L10.59,12L8,14.59L9.41,16L12,13.41L14.59,16L16, 14.59L13.41,12L16,9.41L14.59,8Z" />,
+  'ErrorOutlined',
```

Comment:
![Screenshot_20191226_070016](https://user-images.githubusercontent.com/15232461/71475244-61b07080-27ad-11ea-9ca1-b0bed5cc069f.png)

![Screenshot_20191226_070032](https://user-images.githubusercontent.com/15232461/71475251-6bd26f00-27ad-11ea-9383-23153115c920.png)

I guess this just kinda "is what it is" since this is up to material-design more than anything but it'd be cool if there was a better icon to use here that didn't look like a close icon.  I'm imaging this being used in context, where you'd rarely have more than one alert.

Reply:
The current icons are inspired by https://iamhosseindhv.com/notistack/demos from @iamhosseindhv. Would "ErrorOutiline" be a better fit?

![Capture d’écran 2019-12-26 à 13 12 20](https://user-images.githubusercontent.com/3165635/71475624-6000a000-27e1-11ea-8ae7-4e1deec3ca00.png)

```jsx
import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
```

Reply:
The current icons are inspired by https://iamhosseindhv.com/notistack/demos from @iamhosseindhv. Would "ErrorOutiline" be a better fit?

![Capture d’écran 2019-12-26 à 13 12 20](https://user-images.githubusercontent.com/3165635/71475624-6000a000-27e1-11ea-8ae7-4e1deec3ca00.png)

```jsx
import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
```

Reply:
![Capture d’écran 2019-12-26 à 14 49 24](https://user-images.githubusercontent.com/3165635/71478260-f4253400-27ee-11ea-8ebd-686697da6cfd.png)

@dimitropoulos I think that it looks better. Do you want to replace it?

Reply:
cool: changed in https://github.com/mui-org/material-ui/pull/18702/commits/198e507da5d3f1e3470e2cac6e976b5a7d2ef93e